includes/class-wcs-exporter-cron.php

```
@@ -150,4 +148,35 @@ public static function delete_export_file( $file_name )  {
         }
     }
 
+	/**
+	 * Creates upload directory. Does nothing if directory and index/htaccess files already exist
+	 *
+	 * @since 2.1
+	 * @return void
+	 */
+	protected static function create_upload_directory() {
+		$base = self::$cron_dir;
+
+		if ( wp_mkdir_p( $base ) ) {
+			// create an empty index.php
+			if ( ! file_exists( $base . '/index.php' ) ) {
+				$file_handle = @fopen( $base . '/index.php', 'w' );
```

I am not sure why the @ is used to ignore errors here.